docs/source/faq.rst

```diff
@@ -8,6 +8,8 @@ Optuna is compatible with most ML libraries, and it's easy to use Optuna with th
 Please refer to `examples <https://github.com/pfnet/optuna/tree/master/examples>`_.
 
 
+.. _objective-fun-additional-args:
```

Comment:
I feel 'func' looks more familiar than 'fun' as an abbreviation of function because Python has 'functools' as a built-in module.

Reply:
That makes sense. I replaced "fun" by "func".